views/mixins/recipes.jade

```diff
@@ -282,5 +281,5 @@ mixin thawing()
   div.thawing
     img(src='images/thawing@2x.png', width="49px")
     h6 The main network is thawing
-    p For the first days, while the network defreezes, it is not possible to execute or send transactions, only mining blocks. Please be patient...
+    p For the first days, while the network defreezes, it is not possible to execute or send transactions, only mine blocks. Please be patient...
```

Comment:
Maybe we should mention that they have to download a new version after a while, because the 5000 gas limit is not lifted automatically, and i guess you would get a block limit error when trying to send as its hardcoded